src/Nuget.PackageIndex.VisualStudio/ProjectKFilter.cs

```diff
@@ -43,8 +86,114 @@ public bool IsProjectSupported(string filePath)
                     // TODO Package manager console?
                     Debug.Write(string.Format("{0} \r\n {1}", e.Message, e.StackTrace));
```

Comment:
See the other comment around `Debug.Write`, in NuGet we write such errors to the output. @deepakaravindr might be able to point you to the code.